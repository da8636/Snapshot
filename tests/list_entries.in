SET a 1 2
SET b 5 4
SET c 2 1
LIST ENTRIES
BYE
