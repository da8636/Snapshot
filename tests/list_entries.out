ok

ok

ok

c [2 1]
b [5 4]
a [1 2]

bye
